events/event.html

```diff
@@ -0,0 +1,128 @@
+              </svg>
+              <!-- exit menu icon -->
+              <svg
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                xmlns="http://www.w3.org/2000/svg"
+                class="menu-exit-icon hidden">
+                <path
+                  d="M1.7416 23.9993C1.29575 23.9993 0.849903 23.8293 0.509942 23.4894C-0.169981 22.8094 -0.169981 21.7067 0.509942 21.026L21.0267 0.509942C21.7067 -0.169981 22.8094 -0.169981 23.4901 0.509942C24.17 1.18986 24.17 2.29265 23.4901 2.97327L2.97257 23.4894C2.63261 23.8293 2.18745 23.9993 1.7416 23.9993Z" />
+                <path
+                  d="M22.2577 24C21.8118 24 21.366 23.83 21.026 23.4901L0.509942 2.97327C-0.169981 2.29334 -0.169981 1.19056 0.509942 0.509942C1.18986 -0.169981 2.29265 -0.169981 2.97327 0.509942L23.4901 21.0267C24.17 21.7067 24.17 22.8094 23.4901 23.4901C23.1494 23.83 22.7035 24 22.2577 24Z" />
+              </svg>
+            </button>
+
+            <!-- Page Links -->
+            <ul id="nav-links" class="nav-links">
+              <li><a href="./about.html">About US</a></li>
+              <li><a href="#">Our Impact</a></li>
+              <li><a href="./values.html">Our Values</a></li>
+              <li><a href="./events.html">Events</a></li>
+              <li><a href="./our-team.html">Our Team</a></li>
+              <li><a href="./contact.html">Contact</a></li>
+              <!-- Mobile Support Button -->
+              <li class="mobile-support-btn"><a href="#">Support US</a></li>
+            </ul>
+          </div>
+
+          <!-- desktop support btn -->
+          <a href="#" class="btn-main desktop-support-btn">Support US</a>
+        </div>
+      </div>
+    </nav>
+    <!-- Hero -->
+    <section class="event-detail-container">
+      <img
+        src="/assets/images/pages/event-single/event-poster-WIMW-large.jpg"
+        alt="WIMW Event Poster" />
+      <!-- Wrapper -->
+    </section>
+  </header>
+  <section>
+    <div id="content-container" class="wrapper">
+      <div class="cta-container">
+        <h2>Walking in Multiple Worlds (WiMW)</h2>
+        <p>
+          Walking in Multiple Worlds (WiMW) is an ITLCT program series (consisting of global workshops and speaker sessions starting in 2024) which aims to build awareness around the different ways in which native and indigenous peoples (both in urban and traditional homelands) interrelate and codeswitch in many facets of everyday life and how businesses and organizations can enhance understanding and allyship to build a more inclusive and understanding world (also with digital technology).
+        </p>
+        <a href="#" class="btn-main">View Event Page</a>
```

Comment:
Just wanted to check if this points to an external event page or ITLCT event.html. we should open in a new tab